pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -95,6 +100,35 @@ func GetCloudFromSecret(kubeClient kubernetes.Interface, namespace string, secre
 	return clouds.Clouds[cloudName], nil
 }
 
+func authenticatedClient(ao *gophercloud.AuthOptions) (*gophercloud.ProviderClient, error) {
+
+        client, err := openstack.NewClient(ao.IdentityEndpoint)
+        if err != nil {
+                return nil, err
+        }
+        // add HTTPClient includes transport
+	caFile := "/etc//certs/cacert"
```

Comment:
nit: 2 /

Reply:
Thanks. 
I am sorry I forgot to push so that code was old and lack of some files.